packages/orbit-components/src/utils/Grid/__tests__/index.test.js

```diff
@@ -1,401 +1,110 @@
-    expect(grid).toHaveStyleRule("-ms-grid-row", "1", {
-      media: getBreakpointWidth(QUERIES.LARGEMOBILE, theme),
-      modifier: "& > *:nth-child(2)",
-    });
-    expect(grid).toHaveStyleRule("-ms-grid-column", "1", {
-      media: getBreakpointWidth(QUERIES.LARGEMOBILE, theme),
-      modifier: "& > *:nth-child(3)",
-    });
-    expect(grid).toHaveStyleRule("-ms-grid-row", "2", {
-      media: getBreakpointWidth(QUERIES.LARGEMOBILE, theme),
-      modifier: "& > *:nth-child(3)",
-    });
-    expect(grid).toHaveStyleRule("-ms-grid-column", "3", {
-      media: getBreakpointWidth(QUERIES.LARGEMOBILE, theme),
-      modifier: "& > *:nth-child(4)",
-    });
-    expect(grid).toHaveStyleRule("-ms-grid-row", "2", {
-      media: getBreakpointWidth(QUERIES.LARGEMOBILE, theme),
-      modifier: "& > *:nth-child(4)",
-    });
-  });
-  it("should contain tablet styles", () => {
-    expect(grid).toHaveStyleRule("-ms-grid-columns", "1fr 20px 1fr", {
-      media: getBreakpointWidth(QUERIES.TABLET, theme),
-    });
-    expect(grid).toHaveStyleRule("-ms-grid-rows", "1fr 20px 1fr", {
-      media: getBreakpointWidth(QUERIES.TABLET, theme),
-    });
-  });
-  it("should contain tablet styles for children", () => {
+
     expect(grid).toHaveStyleRule("-ms-grid-column", "1", {
-      media: getBreakpointWidth(QUERIES.TABLET, theme),
       modifier: "& > *:nth-child(1)",
     });
     expect(grid).toHaveStyleRule("-ms-grid-row", "1", {
-      media: getBreakpointWidth(QUERIES.TABLET, theme),
       modifier: "& > *:nth-child(1)",
     });
     expect(grid).toHaveStyleRule("-ms-grid-column", "3", {
-      media: getBreakpointWidth(QUERIES.TABLET, theme),
       modifier: "& > *:nth-child(2)",
     });
     expect(grid).toHaveStyleRule("-ms-grid-row", "1", {
-      media: getBreakpointWidth(QUERIES.TABLET, theme),
       modifier: "& > *:nth-child(2)",
     });
     expect(grid).toHaveStyleRule("-ms-grid-column", "1", {
-      media: getBreakpointWidth(QUERIES.TABLET, theme),
       modifier: "& > *:nth-child(3)",
     });
     expect(grid).toHaveStyleRule("-ms-grid-row", "3", {
-      media: getBreakpointWidth(QUERIES.TABLET, theme),
       modifier: "& > *:nth-child(3)",
     });
     expect(grid).toHaveStyleRule("-ms-grid-column", "3", {
-      media: getBreakpointWidth(QUERIES.TABLET, theme),
       modifier: "& > *:nth-child(4)",
     });
     expect(grid).toHaveStyleRule("-ms-grid-row", "3", {
-      media: getBreakpointWidth(QUERIES.TABLET, theme),
       modifier: "& > *:nth-child(4)",
     });
   });
-  it("should contain desktop styles", () => {
-    expect(grid).toHaveStyleRule(
-      "-ms-grid-columns",
-      "minmax(100px,1fr) 20px minmax(100px,1fr) 20px minmax(100px,1fr) 20px minmax(100px,1fr)",
-      {
-        media: getBreakpointWidth(QUERIES.DESKTOP, theme),
-      },
+});
+
+describe("Grid, different mediaQueries", () => {
+  it.each(Object.entries(QUERIES))("should have styles for each mediaquery: %s", (name, query) => {
```

Comment:
As I said the last time you used this pattern: when I see `it.each` I expect that the component will be rendered differently in every test, but this is not true, the only thing that changes in this test is the assertion at the bottom. Why don't you just do this instead?

```js
  it("should have styles for each media query", () => {
    const dataTest = "test";

    render(
      <Grid
        dataTest={dataTest}
        mediumMobile={{ maxWidth: "600px" }}
        largeMobile={{ maxWidth: "700px" }}
        tablet={{ maxWidth: "800px" }}
        desktop={{ maxWidth: "900px" }}
        largeDesktop={{ maxWidth: "1000px" }}
      >
        <div>1</div>
        <div>2</div>
        <div>3</div>
        <div>4</div>
      </Grid>,
    );

    const expected = {
      mediumMobile: "600px",
      largeMobile: "700px",
      tablet: "800px",
      desktop: "900px",
      largeDesktop: "1000px",
    };

    // eslint-disable-next-line no-restricted-syntax
    for (const query of Object.keys(expected)) {
      expect(screen.getByTestId(dataTest)).toHaveStyleRule("max-width", expected[query], {
        media: getBreakpointWidth(query, theme),
      });
    }
  });
```

Reply:
yes, makes sense